cubedash/summary/_summarise.py

```diff
@@ -167,9 +167,8 @@ def calculate_summary(
                 )
             )
             region_counts = Counter(
-                {
-                    item: count
-                    for item, count in self._engine.execute(
+                dict(
```

Comment:
I don't know Python that well, but this rewrite is not obviously equivalent to me.